README.md

```diff
@@ -1,6 +1,6 @@
 # LeaderBoard
 
-> Hitting up the project API for the Leaderboard list app. You can submit scores and get the scores submitted.
+> This Leaderboard app allows you to get and send scores to an API.
 
 >To run the project download the repository and install all the dependencies.
```

Comment:
-[OPTIONAL] It would be great if you go into the details of how to install the dependencies and start the application, like explaining the need of `npm install` and `npm start`.

Reply:
@leonardodiasb great idea. I would do that.
Thanks